src/Core/LeanCode.Components.Startup/IHostBuilderExtensions.cs

```diff
@@ -15,10 +14,7 @@ public static class IHostBuilderExtensions
     {
         public const string SystemLoggersEntryName = LeanProgram.SystemLoggersEntryName;
 
-        public const string VaultKey = "Secrets:KeyVault:VaultUrl";
-        public const string ClientIdKey = "Secrets:KeyVault:ClientId";
-        public const string ClientSecretKey = "Secrets:KeyVault:ClientSecret";
-        public const string TenantIdKey = "Secrets:KeyVault:TenantId";
+        public const string VaultKey = "KeyVault:VaultUrl";
```

Comment:
We can rename it to `VaultUrlKey` to be more precise. Also I'd keep overrides that allow us to use another key.